memiavl/db.go

```diff
@@ -73,6 +76,9 @@ type Options struct {
 	InitialStores      []string
 	SnapshotKeepRecent uint32
 	SnapshotInterval   uint32
+	// it's more efficient to export snapshot versions, we can filter out the non-snapshot versions
+	SupportExportNonSnapshotVersion bool
```

Comment:
should we make this configurable after this [fix](https://github.com/cosmos/cosmos-sdk/pull/16588/files)?

Reply:
it's mainly a workaround to cope with existing sdk APIs, most user should stick with configs like this:

```
[state-sync]
snapshot-interval = {memiavl.snapshot-interval}

[memiavl]
snapshot-interval = 1000
```

while the `SupportExportNonSnapshotVersion` is `false`.